From: jingi_jeong <jingi_jeong@tmax.co.kr>
Date: Wed, 26 May 2021 16:32:51 +0900
Subject: [feat] desktop-icons-ng : modify unused variable names and add break
 line

- modify unused variable names and add break line

[IMS] N/A
---
 extensions/desktop-icons-ng/desktopManager.js | 22 +++++++++++++---------
 1 file changed, 13 insertions(+), 9 deletions(-)

diff --git a/extensions/desktop-icons-ng/desktopManager.js b/extensions/desktop-icons-ng/desktopManager.js
index 2a5d8d1..f9cebe3 100644
--- a/extensions/desktop-icons-ng/desktopManager.js
+++ b/extensions/desktop-icons-ng/desktopManager.js
@@ -232,7 +232,7 @@ var DesktopManager = class {
             if (item.isSelected) {
                 fileItems.push(item);
                 item.removeFromGrid();
-                let [x, y, unusedA, unusedB, unusedC] = item.getCoordinates();
+                let [x, y, a_, b_, c_] = item.getCoordinates();
                 item.savedCoordinates = [x + deltaX, y + deltaY];
             }
         }
@@ -256,12 +256,12 @@ var DesktopManager = class {
             if (!itemList)
                 return;
 
-            let [x1, y1, unusedX2, unusedY2, unusedC] = this.dragItem.getCoordinates();
+            let [x1, y1, x2_, y2_, c_] = this.dragItem.getCoordinates();
             let oX = x1;
             let oY = y1;
             this._dragList = [];
             for (let item of itemList) {
-                [x1, y1, unusedX2, unusedY2, unusedC] = item.getCoordinates();
+                [x1, y1, x2_, y2_, c_] = item.getCoordinates();
                 this._dragList.push([x1 - oX, y1 - oY]);
             }
         }
@@ -287,7 +287,7 @@ var DesktopManager = class {
         switch (info) {
         case 0:
             if (fileList.length !== 0) {
-                let [xOrigin, yOrigin, unusedA, unusedB, unusedC] = this.dragItem.getCoordinates();
+                let [xOrigin, yOrigin, a_, b_, c_] = this.dragItem.getCoordinates();
                 this.doMoveWithDragAndDrop(xOrigin, yOrigin, xDestination, yDestination);
             }
             break;
@@ -451,12 +451,12 @@ var DesktopManager = class {
             let nemoCustomAtom = Gdk.Atom.intern('x-special/gnome-copied-files', false);
             clipboard.request_contents(nemoCustomAtom, (clipboardIn, selectionData) => {
                 let data = selectionData.get_data();
-                let [valid, unusedAction, unusedFiles] = this._parseClipboardCustomData(data);
+                let [valid, action_, files_] = this._parseClipboardCustomData(data);
                 this._pasteMenuItem.set_sensitive(valid);
             });
         } else {
             clipboard.request_text((clipboardIn, text) => {
-                let [valid, unusedAction, unusedFiles] = this._parseClipboardText(text);
+                let [valid, action_, files_] = this._parseClipboardText(text);
                 this._pasteMenuItem.set_sensitive(valid);
             });
         }
@@ -1335,10 +1335,12 @@ var DesktopManager = class {
 
     doOpenWith(fileItem) {
         let parentWindow;
-        let [x, y, unusedA, unusedB, unusedC] = fileItem.getCoordinates();
+        let [x, y, a_, b_, c_] = fileItem.getCoordinates();
         for (let desktop of this._desktops) {
-            if (desktop._coordinatesBelongToThisGrid(x, y))
+            if (desktop._coordinatesBelongToThisGrid(x, y)) {
                 parentWindow = desktop._window;
+                break;
+            }
         }
 
         let fileItems = this.getCurrentSelection(false);
@@ -1381,8 +1383,10 @@ var DesktopManager = class {
 
         let parentWindow;
         for (let desktop of this._desktops) {
-            if (desktop._coordinatesBelongToThisGrid(X, Y))
+            if (desktop._coordinatesBelongToThisGrid(X, Y)) {
                 parentWindow = desktop._window;
+                break;
+            }
         }
 
         // NOTE(20210526 : jgjeong) : Set parent window because WM determines
